legal-api/tests/unit/resources/v2/test_business_filings/test_filing_documents.py

```diff
@@ -134,7 +135,8 @@ def test_unpaid_filing(session, client, jwt):
     assert rv.json == {}
 
 
-base_url = 'https://LEGAL_API_BASE_URL'
+#base_url = 'https://LEGAL_API_BASE_URL'
+base_url = 'http://localhost:5000'
```

Comment:
Can you revert this back. Test is failing because of this url

Reply:
This is done.